docs/product/issues/issue-details/index.mdx

```diff
@@ -6,29 +6,44 @@ description: "Learn how to navigate the Issue Details page to help you efficient
+
+This page displays the "Details" tab when it first opens, but several other tabs are available depending on issue category:
+
+- **Activity** - A history of activity for the event where team members can share comments.
+- **User Feedback** - Any feedback collected from users through the <PlatformLink to="/user-feedback/">SDK's user feedback</PlatformLink>.
+- **<PlatformLink to="/enriching-events/attachments/">Attachments</PlatformLink>** - Stored additional files, such as config, log or screenshot files that are related to the error event.
+- **Tags** - More detailed information about the tags across all the events included in the issue.
+- **All Events** - A list of all the error events included in the issue.
+- **Merged Issues** - A list of error issues that have been merged with this one.
+- **Similar Issues** - A list of error issues that have a similar stack trace [that you can merge](/concepts/data-management/event-grouping/merging-issues/).
+- **Replays** – A list of [replays](/product/session-replay/) tied to error events included in the issue.
+
 ## Issue Categories & Page Details
 
 There are two categories of issues: [error issues](/product/issues/issue-details/error-issues/) and [performance issues](/product/issues/issue-details/performance-issues/). Depending on the issue category and available event data, the **Issue Details** page displays some combination of the following sections:
 
 - **Event Navigator** - Allows you to select between the latest, oldest, or a recommended issue event and navigate between events in sequence.
 - **Trace Navigator** - An abbreviated view of the related [trace](/product/sentry-basics/tracing/distributed-tracing) for the current transaction.
 - **Suspect Commits** - A commit that's been identified as potentially having caused an error event.
-- **Tags** - Searchable key/value string pairs providing information such as the browser or device.
-- **Screenshot** - Screenshot taken when a user experiences an error, an exception or a crash; only available on [certain SDKs](/product/issues/issue-details/error-issues/#screenshots).
+- **Event Highlights** - Specific [tags](#tags) and [context](#contexts) values that are useful for debugging
 - **Stack Trace** - Shows you the line of code that the event errored on.
 - **Span Evidence** - Information about the performance problem in the context of the current event
 - **Breadcrumbs** - Provide a history and timeline leading up to the error event and are customizable.
-- **Trace Details** - Augments existing error data by capturing interactions between your projects.
+- **Tags** - Searchable key/value string pairs providing information such as the browser or device.
+- **Screenshot** - Screenshot taken when a user experiences an error, an exception or a crash; only available on [certain SDKs](/product/issues/issue-details/error-issues/#screenshots).
+- **Contexts** - Non-searchable key/value data that helps provide context (haha) in debugging the issue.
```

Comment:
```suggestion
- **<PlatformLink to="/enriching-events/context/">Context</PlatformLink>** - Non-searchable key/value data that helps provide context (haha) in debugging the issue.
```
I think it's worth including a link like we do in the product so it's clear where this info comes from. Hope the link is correct!